src/components/content/order/orderStatus/OrderSubmitStatusAlert.tsx

```diff
@@ -103,6 +131,16 @@ function OrderSubmitStatusAlert({
                 stopWatch.pause();
             }
         }
+
+        if (
+            deployedServiceDetails.serviceDeploymentState.toString() ===
+            DeployedServiceDetails.serviceDeploymentState.DEPLOYING.toString()
+        ) {
+            if (!stopWatch.isRunning) {
```

Comment:
we need this as part of the retry request submit button click. As soon as the button is clicked, irrespective of the service status, we must reset the timer and start it again.